extensions/blocks/eventbrite/edit.js

```diff
@@ -47,8 +47,12 @@ class EventbriteEdit extends Component {
 	};
 
 	componentDidMount() {
+		const { setAttributes } = this.props;
 		const { resolvingUrl } = this.state;
 
+		// Use a random number to make the widget div more unique.
+		setAttributes( { blockId: ( new Date().getTime() * Math.random() ).toFixed() } );
+
```

Comment:
Updating the `blockId` attribute on component mount does solve the issue when duplicating a block.

However there is now an unfortunate side effect that the editor detects changes simply when loading the page, and displays a confusing warning when trying to navigate away, even without any content changes.

<img width="462" alt="Screen Shot 2020-02-09 at 20 38 30" src="https://user-images.githubusercontent.com/1699996/74117214-8bb2cc00-4b7c-11ea-9b08-b6da26b6f64b.png">

🤔 if there's a better way to handle this... off the top of my head: maybe there's some other prop or attribute  we can check in `componentDidUpdate` that changes when a block is duplicated but not when the block is loaded in the editor? (I'm not sure if clientId works this way, or not)

Reply:
I'm kinda tempted to make the block `multiple: false` 😄

Reply:
This is actually a very good idea!

Reply:
> This is actually a very good idea!

@scruffian Not really, for at least 2 big reasons! 😄
- Multiple blocks with different events are working fine (and a common enough use case).
- It wouldn't help if the user adds blocks with the same event in different posts that show up in the same archive page.